### PR TITLE
Bug plus misc

### LINKED ISSUE #17: clarify_insert: Make send timers adjustable
Today the `clarify_insert` blocks send data every 5 seconds. This should be adjustable for the user, with a minimum time. Especially if the subscription price differs on number of HTTP request to the Input APi.

### PR DESCRIPTION
**Pull request by khtufteland**
commit b12d8434cdf484b00af3eb07d108ffabb69ee285
Author: Kjetil Hope Tufteland <kjetil@searis.no>
Date:   Mon Mar 29 15:24:18 2021 +0200

    bug: Incorrect form id for clarify_insert name

commit 7c55843e41e4c47d53e7abf1b67d469f1ecd8d7d
Author: Kjetil Hope Tufteland <kjetil@searis.no>
Date:   Mon Mar 29 15:02:41 2021 +0200

    Migrate from momentjs to luxon
    
    Did also change the tokenExpiration text from "Valid until _long_time_"
    till "Expires in 2 hours.".

commit 60e01b5321439a68e6946c157af356fa6dc4e221
Author: Kjetil Hope Tufteland <kjetil@searis.no>
Date:   Mon Mar 29 14:28:03 2021 +0200

    clarify_insert: Left align alwaysEnsure checkbox

commit cc09b5ac8034a5e7d8dcc4e2f912bc140822f39d
Author: Kjetil Hope Tufteland <kjetil@searis.no>
Date:   Mon Mar 29 13:43:48 2021 +0200

    Format with prettier

commit e1e5f088e0be43d6e76127de38404e1bac1488b5
Author: Kjetil Hope Tufteland <kjetil@searis.no>
Date:   Mon Mar 29 13:37:15 2021 +0200

    clarify_insert: Add buffer time as config parameter
    
    Closes #17.

commit d52c96d7f0ff2ff8e8f2bbc994ec402137f94ba1
Author: Kjetil Hope Tufteland <kjetil@searis.no>
Date:   Mon Mar 29 12:58:11 2021 +0200

    clarify_insert: Show warning if alwaysEnsure is active
    
    Closes 20.

commit 4cd5e160ef6190eaa5ebf31e7e4cdce8b54c88a2
Author: Kjetil Hope Tufteland <kjetil@searis.no>
Date:   Mon Mar 29 12:53:57 2021 +0200

    clarify_insert: Move renaming of input fields to advanced options

commit 2453c330e0286035588046ce08d3b246ba94571c
Author: Kjetil Hope Tufteland <kjetil@searis.no>
Date:   Mon Mar 29 12:05:02 2021 +0200

    bug: flush data ignored zeros.
    
    Incorrect validation of data buffer. Values of 0 was skipped. Changed
    it to check if element exists or not.

commit 6c6638c2932004aad298222d1655e15d6145811a
Author: Kjetil Hope Tufteland <kjetil@searis.no>
Date:   Mon Mar 29 11:59:29 2021 +0200

    bug: Change to luxon for datetime parsing
    
    Got a weird bug that was solved by switching to luxon. Added also a
    validation on times.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
         "jwt-decode": "^3.1.2",
         "lodash": "^4.17.21",
         "lowdb": "^1.0.0",
+        "luxon": "^1.26.0",
         "moment": "^2.29.1",
```
Comment (bbergshaven) [inline package.json:41]: do we still need moment?

**Review by bbergshaven — COMMENTED** [review]
Looks like great improvements. One comment on moment.

File: package.json (hunk 1 above)
Comment (khtufteland) [reply package.json:41]: We still use it to check if the access token has expired or not. Could rewrite these parts to use luxon instead and thereby remove the moment dependency. I decided not to do it now, but if you prefer it I don't mind.

Comment (bbergshaven) [reply package.json:41]: One less dependency is always great. Will you do it?

**Review by bbergshaven — APPROVED** [review]

**Pull request merged**